src/main/java/com/exactpro/th2/service/AbstractGrpcService.java

```diff
@@ -64,8 +73,18 @@ protected <T> T createBlockingRequest(Supplier<T> method) {
                 return method.get();
             } catch (StatusRuntimeException e) {
                 exception.addSuppressed(e);
-                if (e.getStatus() == Status.UNKNOWN) { // Server side thrown an exception
-                    throw exception;
+                Status.Code statusCode = e.getStatus().getCode();
+                switch (statusCode) {
```

Comment:
I believe that we should also modify the async version too. Otherwise, we will have a difference in the behavior

Reply:
Thanks, I will correct async behavior